src/pytest/test_replicate.py

```diff
@@ -0,0 +1,52 @@
+import subprocess
+import signal
+import os
+import os.path
+from RLTest import Env
+import time
+
+def testDelReplicate():
+  env = Env(useSlaves=True)
+  master = env.getConnection()
+  slave = env.getSlaveConnection() 
+
+  env.assertContains("PONG", master.execute_command("ping"))
+  env.assertContains("PONG", slave.execute_command("ping"))  
+  env.assertOk(master.execute_command('ft.create', 'idx', 'schema', 'f', 'text'))
+
+  for i in range(10):
+    master.execute_command('ft.add', 'idx', 'doc%d' % i, 1.0, 'fields',
+                                      'f', 'hello world')
+  # Ensure slave is updated
+  master.execute_command('set foo bar')
```

Comment:
This is not the correct way to check it, when we move the replicate to the indexer thread the set might get to the slave before the ft.add command. The correct way is try to get the document from the slave with timeout as I suggested from the first place ...